build-tests/heft-example-plugin-01/.heft/typescript.json

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Why did you add these config files that are entirely commented out?  

For Rush, we add blank templates of everything.  But for Heft, I think we want to minimize that, since otherwise people will copy+paste it as boilerplate to every project in the monorepo.